content/resources/chapter-4/public-key.tsx

```diff
@@ -60,20 +60,18 @@ encoded_private_key = int(0x\${private_key})
 const generatorPoint = G.mul(encoded_private_key)
 # Remember you need to log the answer for it to be validated
 print(generator_point)`,
```

Comment:
```suggestion
return generator_point`,
```

Reply:
@benalleng the python solution for this challenge doesn't appear to be valid - again not your doing but we should probably fix it.

Reply:
had an incorrect case on a param